.travis.yml

```diff
@@ -24,16 +24,17 @@ install:
   - /bin/bash ansible/provision.sh
 
 before_script:
-  - mkdir -p build/logs
+  - mkdir -p build/logs build/coverage
```

Comment:
Based on the new Makefile I suggest we "simplify" the travis.yml file by directly using the make commands (https://github.com/ruflin/Elastica/pull/852). Here for example make prepare